Examples/LocationManager/LocationManager/LocalSearchClient/LocalSearchModels.swift

```diff
@@ -0,0 +1,202 @@
+      && lhs.boundingRegion.span.latitudeDelta == rhs.boundingRegion.span.latitudeDelta
+      && lhs.boundingRegion.span.longitudeDelta == rhs.boundingRegion.span.longitudeDelta
+  }
+}
+
+struct MapItem: Equatable {
+  var isCurrentLocation: Bool
+  var name: String?
+  var phoneNumber: String?
+  var placemark: Placemark
+  var pointOfInterestCategory: MKPointOfInterestCategory?
+  var timeZone: TimeZone?
+  var url: URL?
+
+  init(rawValue: MKMapItem) {
+    self.isCurrentLocation = rawValue.isCurrentLocation
+    self.name = rawValue.name
+    self.placemark = Placemark(rawValue: rawValue.placemark)
+    self.phoneNumber = rawValue.phoneNumber
+    self.pointOfInterestCategory = rawValue.pointOfInterestCategory
+    self.timeZone = rawValue.timeZone
+    self.url = rawValue.url
+  }
+
+  init(
+    isCurrentLocation: Bool = false,
+    name: String? = nil,
+    phoneNumber: String? = nil,
+    placemark: Placemark,
+    pointOfInterestCategory: MKPointOfInterestCategory? = nil,
+    timeZone: TimeZone? = nil,
+    url: URL? = nil
+  ) {
+    self.isCurrentLocation = isCurrentLocation
+    self.name = name
+    self.phoneNumber = phoneNumber
+    self.placemark = placemark
+    self.pointOfInterestCategory = pointOfInterestCategory
+    self.timeZone = timeZone
+    self.url = url
+  }
+
+  static func == (lhs: Self, rhs: Self) -> Bool {
+    lhs.isCurrentLocation == rhs.isCurrentLocation
+      && lhs.name == rhs.name
+      && lhs.phoneNumber == rhs.phoneNumber
+      && lhs.placemark.coordinate.latitude == rhs.placemark.coordinate.latitude
+      && lhs.placemark.coordinate.longitude
+        == rhs.placemark.coordinate.longitude
+      && lhs.placemark.countryCode == rhs.placemark.countryCode
+      && lhs.placemark.region == rhs.placemark.region
+      && lhs.placemark.subtitle == rhs.placemark.subtitle
+      && lhs.placemark.title == rhs.placemark.title
+      && lhs.placemark.name == rhs.placemark.name
+      && lhs.placemark.thoroughfare == rhs.placemark.thoroughfare
+      && lhs.placemark.subThoroughfare == rhs.placemark.subThoroughfare
+      && lhs.placemark.locality == rhs.placemark.locality
+      && lhs.placemark.subLocality == rhs.placemark.subLocality
+      && lhs.placemark.administrativeArea == rhs.placemark.administrativeArea
+      && lhs.placemark.subAdministrativeArea
+        == lhs.placemark.subAdministrativeArea
+      && lhs.placemark.postalCode == rhs.placemark.postalCode
+      && lhs.placemark.isoCountryCode == rhs.placemark.isoCountryCode
+      && lhs.placemark.country == rhs.placemark.country
+      && lhs.placemark.inlandWater == rhs.placemark.inlandWater
+      && lhs.placemark.ocean == rhs.placemark.ocean
+      && lhs.placemark.areasOfInterest == rhs.placemark.areasOfInterest
+      && lhs.pointOfInterestCategory == rhs.pointOfInterestCategory
+      && lhs.timeZone == rhs.timeZone
+      && lhs.url == rhs.url
+  }
+}
+
+struct Placemark: Equatable {
```

Comment:
and here, `MKPlacemark` is not equatable or constructible, so we need a testing friendly version of it.